### DISCUSSION TITLE
HAving a Hard time to udnertsand the project and PRoduct, Any TAkers

### DISCUSSION THREAD

**Discussion by compgeniuses** [DefGuard/defguard#683] [Q&A] [answered]
Hello,

I know what Wiregurad is, and what it does.

I know what SSO is,
I know what MFA/2FA is

But i am having a really hard time to understand this project and or prodcut.
so far there are no videos avaialble describing whats trying to be achieved, and does look like an extremly capable and proising project.

is it a Keycloack/Authentik alternative, as well as WireguardEasy ui alternative that also features a 2FA solution...

SOmeone who has deeper undertsanding kindly help me here.

**Comment by teon** [comment, marked as answer]
I'm no YouTuber and have hard times explaining ,on camera' so baring that in mind here's my attempt to answer you questions:

https://youtu.be/4PF7edMGBwk?si=UoQc91ENFCC_QbZi

Any comments / feedback welcome.

**Reply by compgeniuses** [reply]
Thank you for the Video answer. look into adding it in the main website SMH

**Comment by compgeniuses** [comment]
we need a newer video for 26

**Reply by teon** [reply]
We are now during a **major upgrade to 2.0**.

We will be releasing a series of videos dedicated to new 2.0 and new refreshed setup guides (as setup will change with 2.0 - will **be much easier **).

Stay tuned - release should be ~month or so.
